drivers/Makefile

```diff
@@ -1,12 +1,14 @@
 CC = arm-none-eabi-gcc
-CFLAGS = -c -Wall -I../include/ \
+CFLAGS = -c -Wall -std=gnu11 -Wformat=0 \
```

Comment:
these CFLAGS don't seem ok;
`-std=gnu11` means you are trying to build stuff for a really-new compiler, and ignoring older ones;

no-OS is kind of a generic set of drivers/code; it should probably also run with C89 [for some customers]